<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableEditor</class>
 <widget class="QDialog" name="TableEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Table Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Database Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtDbName">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Table Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtTableName">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Fields:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="tblTables">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>Column Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Data Type</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Length</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Default</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Null Allowed</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Auto Increment</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Primary Key</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="cmdRemoveField">
         <property name="text">
          <string>Remove Field</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cmdInsertField">
         <property name="text">
          <string>Insert Field</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="cmdSaveChanges">
       <property name="text">
        <string>Save Changes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
